scripts/scr_add_corruption/scr_add_corruption.gml

```diff
@@ -1,49 +1,91 @@
-function scr_add_corruption(argument0, argument1) {
+function scr_add_corruption(is_fleet, amount, capital_number,frigate_number, escort_number) 
```

Comment:
I have found that that adding capital_number etc as a a variable declaration to obj_popup (that's where this script is called from) will clear this error as well but this seems to be adding a somewhat random variable to this object that otherwise has little direct interaction with fleets.

All in all this is a minor issue but I have been unable to test this in game so far so I don't know if it will cause issues.

A few notes
1. Ship data in obj.ini is created on starting a new game (this does not include capital_number etc)
2. Alarm 1 in obj_controller creates obj_p_fleet and reads the ship data in obj.ini to populate it and sets capital_number and some other vars in obj_p_fleet but not in obj.controller.
3. When examining the code for the script the capital_number etc is undefined where the repeat loops expect a real.
4. We need to refer to the capital_number etc variables in obj_p_fleet as we can have multiple instances of this and each may need to run the script these variables are not held in obj.ini or obj.controller as far as I can determine.